src/CoreGraphics/CGAffineTransform.cs

```diff
@@ -88,13 +90,16 @@ public struct CGAffineTransform {
 		[Obsolete ("Use 'Ty' instead.")]
 		public /* CGFloat */ nfloat y0;   // ty
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
I don't know how people feel about the individual warnings, but I'm thinking just add it once per file around a #If NET rather than making so much noise in the diff.

Reply:
I slightly prefer once per file but don't care enough either way.